workers/unity/Assets/Fps/Scripts/UI/Controllers/Testers/LobbyScreenTester.cs

```diff
@@ -1,51 +1,38 @@
-﻿using UnityEngine;
+﻿using System.Collections;
+using System.Collections.Generic;
+using Fps;
+using UnityEngine;
 
-namespace Fps
+public class LobbyScreenTester : MonoBehaviour
```

Comment:
Needs namespace

Reply:
Also not really testing something - but providing mock data.

Reply:
I intend all these Tester scripts to be removed before going into `/develop`, but noted.

Reply:
> Needs namespace

👍